src/main/kotlin/com/katanox/tabour/core/EventConsumer.kt

```diff
@@ -1,16 +1,16 @@
 package com.katanox.tabour.core
 
-import com.katanox.tabour.base.IEventConsumerBase
+import com.katanox.tabour.base.IEventHandlerBase
 import com.katanox.tabour.factory.BusType
-import com.katanox.tabour.factory.EventConsumerFactory
+import com.katanox.tabour.factory.EventHandlerFactory
 import org.springframework.beans.factory.annotation.Autowired
 import javax.annotation.PostConstruct
 
 abstract class EventConsumer {
 
-    private lateinit var consumerBase: IEventConsumerBase
+    private lateinit var handlerBase: IEventHandlerBase
```

Comment:
i get a bit confused with the naming.
I think`eventHandler` or just `handler` would indicate what this variable is

Reply:
Do you mean the handlerBase to be renamed?